doc/ChangeLog

```diff
@@ -1,4 +1,98 @@
 ===============================================================
+Tag name:  ctsm1.0.dev017
+Originator(s):  slevis (Samuel Levis, Slevis Consulting LLC,303-665-1310)
+Date:  Fri Nov 16 14:54:37 MST 2018
+One-line Summary: Merge the collapse2gencrop branch
+
+Purpose of changes
+------------------
+
+These changes allow the model to not need to read 16-pft
+datasets and rather read 78-pft datasets. The 78-pft datasets were
+developed for use with prognostic crops originally. The current changes
+allow the model to use the 78-pft datasets in 16-pft runs by
+collapsing the crop pfts (cfts) from specific types to the model's
+generic crop types. The changes are generic so that the model may still
+read 16-pft datasets for 16-pft runs. Ultimately these changes will
+permit users to run with any number of pfts, while the ctsm group
+maintains a single set of input pft data.
+
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+[ ] clm5_0
+
+[ ] clm4_5
+
+[ ] clm4_0
+
+Notes of particular relevance for users
+---------------------------------------
+
+Changes made to namelist defaults (e.g., changed parameter values):
+maxpatch_pft made obsolete
+
+Substantial timing or memory changes:
+Cheyenne gnu 7 TPUTCOMP FAILs
+Cheyenne intel 4 TPUTCOMP & 8 MEMCOMP FAILs
+Hobart gnu 1 TPUTCOMP FAIL
+Hobart nag 4 TPUTCOMP FAILs
+
+Notes of particular relevance for developers: (including Code reviews and testing)
```

Comment:
Also add a note that we should remove maxpatch_pft from the code and namelist as a cleanup step. Currently it's still there, but should be removed.

Reply:
DONE.